cmd/generate_repo_config/generate_repo_config_test.go

```diff
@@ -66,7 +67,7 @@ def go_repositories():
 	}
 	defer os.RemoveAll(tmp)
 
-	if err := generateRepoConfig(tmp+"/WORKSPACE", dir+"/WORKSPACE"); err != nil {
+	if err := generateRepoConfig(filepath.Join(tmp, "/WORKSPACE"), filepath.Join(dir, "/WORKSPACE")); err != nil {
```

Comment:
Don't include leading slashes in arguments to `filepath.Join`, other than the first argument. `filepath.Join` adds path separators (`/` or `\`, depending on platform) between arguments and converts interior slashes to path separators. This might work sometimes, but I'm not sure it will always work, especially on Windows.